atf_eregs/settings/dev.py

```diff
@@ -3,6 +3,18 @@
 DEBUG = True
 TEMPLATE_DEBUG = DEBUG
 
+# Analytics settings
+ANALYTICS = {
+    'GOOGLE': {
+       'GTM_SITE_ID': '',
+       'GA_SITE_ID': '',
+    },
+    'DAP': {
+        'AGENCY': '',
+        'SUBAGENCY': '',
+    },
+}
```

Comment:
Is this setting needed anymore? It should be imported from regulations-site automatically, hey?